### PR TITLE
Add physical_mode TRAM to Navitia Providers

### PR DESCRIPTION
**Pull request by jamescr**
This PR adds support for the navitia physical_mode=TRAM as described in the issue  #144

### CODE REVIEW

File: enabler/src/de/schildbach/pte/AbstractNavitiaProvider.java [hunk 1]
```diff
@@ -92,8 +92,7 @@
     }
 
     private enum PhysicalMode {
-        AIR, BOAT, BUS, BUSRAPIDTRANSIT, COACH, FERRY, FUNICULAR, LOCALTRAIN, LONGDISTANCETRAIN, METRO, RAPIDTRANSIT, SHUTTLE, TAXI, TRAIN, TRAMWAY, OTHER
-    }
+        AIR, BOAT, BUS, BUSRAPIDTRANSIT, COACH, FERRY, FUNICULAR, LOCALTRAIN, LONGDISTANCETRAIN, METRO, RAPIDTRANSIT, SHUTTLE, TAXI, TRAIN, TRAMWAY, TRAM, OTHER    }
```
Comment (schildbach) [inline enabler/src/de/schildbach/pte/AbstractNavitiaProvider.java]: Please don't change the position of the closing brace.

File: enabler/src/de/schildbach/pte/AbstractNavitiaProvider.java [hunk 2]
```diff
@@ -199,7 +198,7 @@ private LocationType getLocationType(PlaceType placeType) {
     /**
      * Some Navitia providers return location names with wrong case. This method can be used to fix the name
      * when locations are parsed.
-     * 
+     *
```
Comment (schildbach) [inline enabler/src/de/schildbach/pte/AbstractNavitiaProvider.java]: This is an unrelated change.

**Comment by jamescr** [comment]
> Can you explain why you mapped TRAM to SUBURBAN_TRAIN rather than TRAM?

I got confused between the differences between the types of trains... clearly it should be mapped to TRAM.

**Comment by schildbach** [comment]
Merged, thanks.
